### PR TITLE
Add tick svg to summary page once a user publishes a listing

### PR DESCRIPTION
**Pull request by cpjmcquillan**
## Jira ticket URL:
[TEVA-547](https://dfedigital.atlassian.net/browse/TEVA-547)
[Figma](https://www.figma.com/file/n2wLtKVQ2imMbjdPSHBRKW/Sprint-50?node-id=0%3A1
)
[Prototype](https://dfe-prototype.herokuapp.com/hs_2/8-success
)
## Changes in this PR:
- Add the tick svg

image

### CODE REVIEW

**Comment by david-mears-dfe** [comment]
This is probably more UX's job to point out, but it looks to me on the Figma that there's equal distance above and below the tick, within the dark green box. On the screenshot it looks a little low.

**Comment by cpjmcquillan** [comment]
> This is probably more UX's job to point out, but it looks to me on the Figma that there's equal distance above and below the tick, within the dark green box. On the screenshot it looks a little low.

The font size is also different between Figma and Prototype. I've copied the prototype since the Prototype

**Comment by david-mears-dfe** [comment]
> > This is probably more UX's job to point out, but it looks to me on the Figma that there's equal distance above and below the tick, within the dark green box. On the screenshot it looks a little low.
> 
> The font size is also different between Figma and Prototype. I've copied the prototype since the Prototype

I think your wifi connection broke up there. Say again?

File: app/frontend/styles/components/hiring-staff.scss [hunk 1]
```diff
@@ -25,3 +25,32 @@ body.hiring-staff {
   margin-bottom: 0;
   margin-top: 1.5rem;
 }
+
+.hod-checkmark {
```
Comment (david-mears-dfe) [inline app/frontend/styles/components/hiring-staff.scss:29]: what is '.hod'?

**Pull request merged**